Source/settings/mods.js

```diff
@@ -39,11 +39,11 @@ var FullScreenPokemon;
                 events: {
                     onModEnable: function (mod) {
                         var stats = this.ObjectMaker.getFunction("Player").prototype;
-                        this.player.speed = stats.speed = 10;
+                        stats.speed = 10;
                     },
                     onModDisable: function (mod) {
                         var stats = this.ObjectMaker.getFunction("Player").prototype;
-                        this.player.speed = stats.speed = this.settings.objects.properties.Player.speed;
```

Comment:
You'll have to not change these lines, they're a bug fix from Blake.

https://github.com/FullScreenShenanigans/FullScreenPokemon/blame/mods/Source/settings/mods.ts#L45